src/app/components/email-validation/index.html

```diff
@@ -1,27 +1,115 @@
-<sky-demo-page
+<sky-tabbed-demo-page
   pageTitle="Email Validation"
-  navTitle="Email validation"
-  skyuxModule="SkyEmailValidationModule"
-  packageName="@skyux/validation"
 >
 
-  <sky-demo-page-summary>
-    The email validation directive ensures that user entries in an <sky-code>input</sky-code> element are valid email addresses.
-  </sky-demo-page-summary>
+<sky-demo-page-summary>
+The email validation directive validates the format of email addresses in input fields.
+</sky-demo-page-summary>
 
-  <sky-demo-page-properties sectionHeading="Email validation input directive properties">
-    <sky-demo-page-property
-      propertyName="skyEmailValidation"
-    >
-      Creates an input that validates email addresses. Place this attribute on an <sky-code>input</sky-code> element. If users enter values that are not valid email addresses, an error message appears. The directive uses <sky-code>NgModel</sky-code> to bind data.
-    </sky-demo-page-property>
-  </sky-demo-page-properties>
+<div class="sky-margin-stacked-content-section">
+    <sky-demo-page-example>
+        <sky-email-validation-demo></sky-email-validation-demo>
+    </sky-demo-page-example>
+</div>
+    
+<sky-tabset id="content-tabs">
+	
+	<sky-tab tabHeading="Design">
+        
+        <sky-demo-page-content sectionHeading="Usage">
+            <h3 class="sky-text-success">Use when</h3>
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use email validation when users must enter valid email addresses.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/email-validation/email-validation-usage-1.png" />
+                    <div class="caption caption-success sky-text-success">Do validate email addresses that users will store as data.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+                
+            </sky-docs-visual-example>
+        
+            <h3 class="sky-text-danger">Don't use when</h3>
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                 <p>Don’t use email validation when users can enter multiple types of data in an input field.</p>
```

Comment:
The "Don't" usage might be overkill; feels like we could get away with only having the "Do" usage.